src/EventListener/CacheControlListener.php

```diff
@@ -56,7 +58,7 @@ public function __construct(
          * @var string|false Name of the header or false to add no header
          */
         private readonly string|false $debugHeader = false,
-        private readonly string $ttlHeader = 'X-Reverse-Proxy-TTL',
```

Comment:
This is kinda unnecessary because always set explicitly, but does not hurt.

Reply:
it is relevant for BC in case somebody configured their own service with the class.